src/http/server.ts

```diff
@@ -167,38 +168,41 @@ export class Server {
   // FILE MARKER - PRIVATE METHODS /////////////////////////////////////////////
   //////////////////////////////////////////////////////////////////////////////
 
-  #getHandler(): (r: Request) => Promise<Response> {
+  #getHandler(): (r: Request, connInfo: ConnInfo) => Promise<Response> {
     const resources = this.#resources;
     const serverServices = this.#options.services ?? [];
-    return async function (originalRequest: Request): Promise<Response> {
+    return async function (
+      originalRequest: Request,
+      connInfo: ConnInfo,
+    ): Promise<Response> {
+      // Grab resource and path params
+      const resourceAndParams = getResourceAndParams(
+        originalRequest.url,
+        resources,
+      ) ?? {
+        resource: null,
+        pathParams: new Map(),
+      };
+      const { resource, pathParams } = resourceAndParams;
+
+      // Construct request and response objects to pass to services and resource
+      const request = await Drash.Request.create(
```

Comment:
instead of having the service's methods take in `ConnInfo`, can we add the `ConnInfo` to our request object and then have it be accessible like `request.conn_info` via getter or public property?

Reply:
like we wouldn't need this:

```typescript
  public runAfterResource(
    _request: Request,
    _response: Response,
    _connInfo: ConnInfo,
  ) {
  }
```

we could just keep `request` and `response` and the `request` would have the `conn_info` as a property or something.

Reply:
sgtm